wyckoff_generation/common/registry.py

```diff
@@ -1,5 +1,23 @@
 """
-This code was borrowed and modified from https://github.com/FAIR-Chem/fairchem/blob/main/src/fairchem/core/common/registry.py
+This code was borrowed and modified from https://github.com/FAIR-Chem/fairchem/blob/main/src/fairchem/core/common/registry.py which is licensed under an MIT License
+See license text below:
```

Comment:
Lite omformulering, och vi bör nog använda en specifik commit här. Jag tog den senaste som modifierade den här filen, men du vill kanske kolla om du utgick ifrån någon äldre.

```suggestion
This file is based on code made available under the MIT license by the fairchem repository by FAIR Chemistry available from their repository as https://github.com/facebookresearch/fairchem/blob/47b189b394ba90b9117ec68bb2074cf037984f02/src/fairchem/core/common/registry.py

Modifications Copyright (c) 2025 The High-Throughput Toolkit

The original copyright header from the fairchem file follows below:
```